torch/nn/modules/batchnorm.py

```diff
@@ -55,6 +55,7 @@ def __init__(
             self.register_buffer('num_batches_tracked',
                                  torch.tensor(0, dtype=torch.long,
                                               **{k: v for k, v in factory_kwargs.items() if k != 'dtype'}))
+            self.num_batches_tracked: Optional[Tensor]
```

Comment:
With this new annotation, you'll need to make a change like `self.num_batches_tracked = 1  # type: ignore[assignment]` in `torch/distributed/pipeline/sync/batchnorm.py` to pass mypy checks.

(you can run `mypy` locally to check for errors)

Reply:
I've instead added the annotation `num_batches_tracked: Tensor` in `torch/distributed/pipeline/sync/batchnorm.py`, which passes `mypy` and seems correct to me. Let me know if this is incorrect or inadvisable, though...